cmd/kops/create_cluster.go

```diff
@@ -575,6 +580,18 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 		}
 	}
 
+	if c.MasterVolumeSize != "" {
+		for _, group := range masters {
```

Comment:
Right -- as far as i understand it, it gets hardcoded in the autoscaling policy creation. The thought was to simply defer to that if its unset.